### ISSUE TITLE
Particles cause massive lag in basalt deltas

### ISSUE THREAD

**Issue by NolanHewitt** [jellysquid3/Sodium#2425]
### Bug Description

Frame rates drop from 100fps down to 10-15fps by switching between particles on or off in basalt deltas. This is caused by the dust and white ash particles. Just turning those two off with sodium extra also eliminates the lag.

### Reproduction Steps

Enter basalt delta biome, game begins to lag.

### Log File

[latest.log](https://github.com/CaffeineMC/sodium-fabric/files/15047011/latest.log)

### Crash Report

[crash-2024-04-20_00.56.48-client.txt](https://github.com/CaffeineMC/sodium-fabric/files/15047010/crash-2024-04-20_00.56.48-client.txt)

**Comment by MeeniMc** [comment]
You have a massive modpack, try to reproduce with Sodium only. This is a bug tracker mostly for developpers, you may try to join the discord to receive user support.

**Comment by NolanHewitt** [comment]
After more testing I'm left confused. I removed all mods except sodium and the lag was gone, so I assumed it must be some other mod instead. I started slowly adding more and more mods, testing the world each few mods added, but the lag didn't return. Eventually with all the mods I thought I found the cause, distant horizons, as in a new world the lag returned, but the lag went away when I disabled distant generation and disabled rendering, but after turning them back on the lag didn't return. So I had all my mods and DH enabled in a new world but the lag was also gone? I made yet another world and still, I went to a basalt delta and the lag remained gone. This issue with basalt deltas lagging has plagued my for months so I have no idea what is going on.
